### PR TITLE
cloud: add api overview

### PR DESCRIPTION
**Pull request by ran-huang**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add Cloud API overview

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: TOC-tidb-cloud.md [hunk 1]
```diff
@@ -24,6 +24,7 @@
     - [Statement Analysis](/tidb-cloud/tune-performance.md)
     - [Key Visualizer](/tidb-cloud/tune-performance.md#key-visualizer)
   - [Upgrade a TiDB Cluster](/tidb-cloud/upgrade-tidb-cluster.md)
+  - [Use Open API](/tidb-cloud/open-api-overview.md)
```
Comment (ran-huang) [inline TOC-tidb-cloud.md:27]:
```suggestion
  - [Use TiDB Cloud API](/tidb-cloud/open-api-overview.md)
```

TODO: add redoc link to reference

File: tidb-cloud/api-overview.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+The API has the following features:
+
+- **JSON entities.** All entities are expressed in JSON.
+- **HTTPS-only.** You can only access the API via HTTPS, ensuring all the data sent over the network is encrypted with TLS.
+- **Key-based access and digest authentication.** Before you access TiDB Cloud API, you must generate an API key. All requests are authenticated through [HTTP Digest Authentication](https://en.wikipedia.org/wiki/Digest_access_authentication), ensuring the API key is never sent over the network.
+
+To start using TiDB Cloud API, refer to the following resources:
+
+- [Get Started](https://docs.pingcap.com/tidbcloud/api/v1#section/Get-Started)
```
Comment (Oreoxmt) [inline tidb-cloud/api-overview.md:13]:
```suggestion
- [Get Started](https://docs.pingcap.com/tidbcloud/api/v1#section/Get-Started)
- [Authentication](https://docs.pingcap.com/tidbcloud/api/v1#section/Authentication)
- [Rate Limiting](https://docs.pingcap.com/tidbcloud/api/v1#section/Rate-Limiting)
```

File: tidb-cloud/api-overview.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# TiDB Cloud API Overview
+
+The TiDB Cloud API is a [REST interface](https://en.wikipedia.org/wiki/Representational_state_transfer) that provides you with programmatic access to manage administrative objects within TiDB Cloud, including projects, cluster deployments, and backups.
```
Comment (ran-huang) [inline tidb-cloud/api-overview.md:3]:
```suggestion
The TiDB Cloud API is a [REST interface](https://en.wikipedia.org/wiki/Representational_state_transfer) that provides you with programmatic access to manage administrative objects within TiDB Cloud. Through this API, you can manage resources automatically and efficiently, including:

* Projects
* Clusters
* Backups
* Restores
```

File: TOC-tidb-cloud.md [hunk 4]
```diff
@@ -28,6 +28,7 @@
     - [Compatibility](/tiflash/tiflash-compatibility.md)
   - [Scale a TiDB Cluster](/tidb-cloud/scale-tidb-cluster.md)
   - [Upgrade a TiDB Cluster](/tidb-cloud/upgrade-tidb-cluster.md)
+  - [Use TiDB Cloud API](/tidb-cloud/open-api-overview.md)
```
Comment (Oreoxmt) [inline TOC-tidb-cloud.md:31]:
```suggestion
  - [Use TiDB Cloud API](/tidb-cloud/api-overview.md)
```

**Review by qiancai — APPROVED** [review]

File: TOC-tidb-cloud.md [hunk 5]
```diff
@@ -344,6 +345,7 @@
         - [`USER_PRIVILEGES`](/information-schema/information-schema-user-privileges.md)
         - [`VIEWS`](/information-schema/information-schema-views.md)
   - [System Variables](/system-variables.md)
+  - [API Reference](https://docs.pingcap.com/tidb-cloud/api/v1)
```
Comment (qiancai) [inline TOC-tidb-cloud.md:348]:
```suggestion
  - [API Reference](https://docs.pingcap.com/tidbcloud/api/v1)
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: tidb-cloud/api-overview.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
+# TiDB Cloud API Overview
```
Comment (lilin90) [inline tidb-cloud/api-overview.md:1]: Please add the metadata for this file.

File: TOC-tidb-cloud.md [hunk 7]
```diff
@@ -28,6 +28,7 @@
     - [Compatibility](/tiflash/tiflash-compatibility.md)
   - [Scale a TiDB Cluster](/tidb-cloud/scale-tidb-cluster.md)
   - [Upgrade a TiDB Cluster](/tidb-cloud/upgrade-tidb-cluster.md)
+  - [Use TiDB Cloud API](/tidb-cloud/api-overview.md)
```
Comment (lilin90) [inline TOC-tidb-cloud.md:31]: Also add these two API links to `_index.md`?

File: tidb-cloud/_index.md [hunk 8]
```diff
@@ -72,6 +73,7 @@ aliases: ['/tidbcloud/beta/privacy-policy', '/tidbcloud/beta/terms-of-service',
 <ColumnTitle>Reference</ColumnTitle>
 
 - [Billing](/tidb-cloud/tidb-cloud-billing.md)
+- [API References](/tidb-cloud/api-reference.md)
```
Comment (lilin90) [inline tidb-cloud/_index.md:76]:
```suggestion
- [API Reference](https://docs.pingcap.com/tidbcloud/api/v1)
```

**Review by lilin90 — APPROVED** [review]
LGTM

**Pull request merged**